### PR TITLE
test(pubsub): add tracing enabled tests for blocking publisher

### PR DESCRIPTION
**Pull request by alevenberg**
Part of https://github.com/googleapis/google-cloud-cpp/issues/13250

- - -
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/13251)

### CODE REVIEW

**Comment by alevenberg** [comment]
> > use old admin client
> 
> Q. Why is that a feature?

Not a feature... maybe a bug. It builds with the handwritten admin api but not the generated one.

**Comment by coryan** [comment]
> > > use old admin client
> > 
> > 
> > Q. Why is that a feature?
> 
> Not a feature... maybe a bug. It builds with the handwritten admin api but not the generated one.

Maybe this:

https://github.com/googleapis/google-cloud-cpp/blob/c4633e735d7277cd18c6b0babf9160c6f76c329e/generator/generator_config.textproto#L2603-L2621

is missing one of these?

https://github.com/googleapis/google-cloud-cpp/blob/c4633e735d7277cd18c6b0babf9160c6f76c329e/generator/generator_config.textproto#L2628

**Comment by alevenberg** [comment]
Gonna revert the commit on https://github.com/googleapis/google-cloud-cpp/pull/13241, and update the PR after this is merged

File: generator/generator_config.textproto [hunk 1]
```diff
@@ -2605,6 +2605,7 @@ service {
   product_path: "google/cloud/pubsub/admin"
   initial_copyright_year: "2023"
   retryable_status_codes: ["kUnavailable"]
+  emulator_endpoint_env_var: "PUBSUB_EMULATOR_HOST"
```
[the remark this answers is not in the document]
Comment (alevenberg) [reply generator/generator_config.textproto:2608]: This one's outdated. Sorry.

**Pull request merged**